courses/free-courses-mr.md

```diff
@@ -0,0 +1,85 @@
+### Index
+
+* [Android](#android)
+* [Angular](#angular)
+* [Arduino](#arduino)
+* [C](#c)
+* [C#](#csharp)
+* [Data Science](#datascience)
+* [Databases](#databases)
+* [HTML and CSS](#html-and-css)
+* [Java](#java)
+* [JavaScript](#javascript)
+* [PLC](#plc)
+* [Python](#python)
+
+
+### Android
+
+* [अँड्रॉइड ऍप डेव्हलपमेंट मराठी \| Android App Development in Marathi](https://youtube.com/playlist?list=PLy1NOWTMDFjDLCgmxRNLDJtlutJOYl2d7) - उत्तर
+
+
+### Angular
+
+* [Angular Tutorials in Marathi](https://youtube.com/playlist?list=PLMs908ICeVMEBw4CevNdPvvIJDaQV1ISN) - Developers Marathi
```

Comment:
sorry, one more nit. could you switch the order of the Angular course so it's alphabetical? thanks